src/c_api/c_api.cc

```diff
@@ -114,21 +114,21 @@ int MXNotifyShutdown() {
   API_END();
 }
 
-int MXSetTracerOutput(int enable, const char* filename) {
+int MXSetProfilerOutput(int enable, const char* filename) {
   API_BEGIN();
-  engine::Tracer::Get()->SetOutput(enable != 0, std::string(filename));
+  engine::Profiler::Get()->SetOutput(enable != 0, std::string(filename));
   API_END();
 }
 
-int MXTurnTracerOn() {
+int MXTurnProfilerOn() {
```

Comment:
minimize the API by allowing pass of function and combine on and off together

MXSetProfilerState